change/@fluentui-react-context-selector-d5e34afc-03aa-408c-8192-bb66d5b01f19.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32207&commitId=f2a38a330e7b2465ab92669d265a532e21c835e3&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|3| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|4| Changed|
|Avatar Converged.basic - High Contrast.normal.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>